src/components/QueryOutput/QueryOutput.js

```diff
@@ -2313,6 +2419,9 @@ export class QueryOutput extends React.Component {
           queryFn={this.queryFn}
           onBucketSizeChange={this.props.onBucketSizeChange}
           bucketSize={this.props.bucketSize}
+          queryResponse={this.queryResponse}
```

Comment:
Do we need to pass this in anymore? It is a ton of data and might slow down performance if its not needed

Reply:
nope, I forgot to delete it, thank you for reminding me!